src/dashboard/src/components/filesystem_ajax/views.py

```diff
@@ -549,14 +551,14 @@ def copy_from_arrange_to_completed_common(filepath, sip_uuid, sip_name):
             ).split("/", 1)
             transfer_name = transfer_parts[0]
             # Determine if the transfer is a BagIt package
-            is_bagit = True if transfer_parts[1].startswith(u"data/objects") else False
+            is_bagit = transfer_parts[1].startswith(u"data/")
             # Copy metadata & logs to tmp/, where later scripts expect
             for directory in ("logs", "metadata"):
-                source = [DEFAULT_BACKLOG_PATH, transfer_name, directory]
+                source = [DEFAULT_BACKLOG_PATH, transfer_name, directory, "."]
```

Comment:
Not suggesting to change it now, but are these trailing dots necessary to turn the paths directories? I think I've seen (and used) spaces instead.

Reply:
I was assuming this is what SS was expecting, isn't it?